drivers/Makefile.include

```diff
@@ -103,3 +103,6 @@ endif
 ifneq (,$(filter veml6070,$(USEMODULE)))
     USEMODULE_INCLUDES += $(RIOTBASE)/drivers/veml6070/include
 endif
+ifneq (,$(filter uart_half_duplex,$(USEMODULE)))
```

Comment:
I don't want the header to be in the common include dir because I don't want users to use this interface. They can use it if they want anyway, but I put it there to tell it's not intended to be used.
Only drivers like feetech and dynamixel should use it.